frame/transaction-payment/src/lib.rs

```diff
@@ -1540,6 +1540,36 @@ mod tests {
 		});
 	}
 
+	#[test]
+	fn one_tip_has_more_priority() {
```

Comment:
Has more than what? :P 

I would find it better if you compare this against 0 here.